iphone/Maps/Core/Framework/ProxyObjects/Routing/MWMRoutingManager.mm

```diff
@@ -250,7 +250,8 @@ - (void)speedCameraLeftVisibleArea {
 - (void)onLocationUpdate:(CLLocation *)location {
   NSMutableArray<NSString *> * turnNotifications = [NSMutableArray array];
   std::vector<std::string> notifications;
-  self.rm.GenerateNotifications(notifications);
+  auto announceStreets = [NSUserDefaults.standardUserDefaults boolForKey:@"UserDefaultsNeedToEnableStreetNamesTTS"];
```

Comment:
```suggestion
  bool const announceStreets = [NSUserDefaults.standardUserDefaults boolForKey:kIsStreetNamesTTSEnabled];
```

Reply:
In the beginning of the file, above @interface, add
```
extern NSString * const kIsStreetNamesTTSEnabled;
```

Reply:
It is a pointer type, not an integral type. The difference is that `const` marked variables can be placed in the read-only memory sections of the executable, which may lead to better performance (no guards when accessing), lesser code size, and more safety against accidental modifications.